### ISSUE TITLE
[Sev2] Color contrast ratio is less than 4.5:1 for the time range tooltip under 'new dashboard' screen

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#117284]
User Impact:
Low vision users will face difficulty in reading the text and understanding the page content if the color contrast for any control or text is insufficient i.e. less than minimum required ratio 4.5:1.

**Steps to reproduce**:
1. Create a 'New dashboard'
2. Navigate to 'Time Range' control and mouse hover on it
3. Verified whether color contrast ratio of the tooltip text is less than the 4.5:1 or not

**Actual Result**:
Color contrast ratio of the Time Range tooltip text is 2.5:1 which is less than the 4.5:1 under 'New dashboard' screen.

**Expected Result**
Color contrast ratio of the Time Range tooltip text should be equal to or more than 4.5:1 under 'New dashboard' screen.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
WCAG: [1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
Section 508: [E207.2 WCAG Conformance](https://www.access-board.gov/ict/#E207.2), [E205.4 Accessibility Standard](https://www.access-board.gov/ict/#E205.4) and [E205.4.1 Word Substitution when Applying WCAG to Non-Web Documents](https://www.access-board.gov/ict/#E205.4.1), and [E207.2.1 Word Substitution when Applying WCAG to Non-Web Software](https://www.access-board.gov/ict/#e207-2-1-word-substitution-when-applying-wcag-to-non-web-softwar).
EN 301 549: [PDF of EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), which applies WCAG principles for Contrast Minimum to 9.1.4.3 web, 10.1.4.3 non-web documents, and 11.1.4.3 software.

**Environment**:
- Grafana version: 12.2.0
- Data source type & version: n/a
- User OS & Browser: any
- Others:
